mlflow/server/js/src/experiment-tracking/pages/experiment-overview/utils/chartUtils.ts

```diff
@@ -0,0 +1,41 @@
+import { TIME_BUCKET_DIMENSION_KEY, type MetricDataPoint } from '@databricks/web-shared/model-trace-explorer';
+
+/**
+ * Format timestamp for x-axis labels based on time interval granularity
+ * @param timestampMs - Timestamp in milliseconds
+ * @param timeIntervalSeconds - Time interval in seconds used for grouping
+ */
+export function formatTimestamp(timestampMs: number, timeIntervalSeconds: number): string {
```

Comment:
can we update this name to be something more specific e.g. `formatTimestampForRecharts`? We already have various `formatTimestamp` methods [here](https://sourcegraph.prod.databricks-corp.com/mlflow/mlflow/-/blob/mlflow/server/js/src/common/utils/Utils.tsx?L134) and [there](https://sourcegraph.prod.databricks-corp.com/databricks-eng/universe/-/blob/js/packages/web-shared/src/genai-traces-table/utils/MlflowUtils.tsx?L551) so it's best to avoid more confusion